### PR TITLE
roster bot

### PR DESCRIPTION
**Pull request by tsuwandy**
Fixes #
https://github.com/microsoft/botbuilder-js/issues/1211
## Description

I can't test this as TeamsActivityHandler is still missing roster methods (added as placeholders for now in the bot class).

## Specific Changes

  - 
  -
  -

### CODE REVIEW

File: libraries/botbuilder/tests/teams/roster/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "messaging-extension-action",
```
Comment (daveta) [inline libraries/botbuilder/tests/teams/roster/package.json]: >messaging-extension-action [](start = 11, length = 26)

May want to rename for clarity..

File: libraries/botbuilder/tests/teams/roster/src/rosterBot.ts [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    }
+   
+    private async showDetails(context: TurnContext): Promise<void> {
+        let teamDetails = await this.getTeamDetails(context);
+        var replyActivity = this.createReply(context.activity, `The team name is ${teamDetails.Name}. The team ID is ${teamDetails.Id}. The AAD GroupID is ${teamDetails.AadGroupId}.`);
+        await context.sendActivity(replyActivity);
+    }
+
+    private async getMembers(context: TurnContext): Promise<any[]> {
+        return null;
```
Comment (daveta) [inline libraries/botbuilder/tests/teams/roster/src/rosterBot.ts]: >null [](start = 15, length = 4)

Assuming these are stubbed until in RosterClient completed..

File: libraries/botbuilder/tests/teams/roster/teams-app-manifest/manifest.json [hunk 3]
```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.3/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.3",
+  "version": "1.0.0",
+  "id": "",
+  "packageName": "com.teams.sample.echobot",
```
Comment (daveta) [inline libraries/botbuilder/tests/teams/roster/teams-app-manifest/manifest.json]: >echobot [](start = 35, length = 7)

For fun I change this.

File: libraries/botbuilder/tests/teams/roster/src/rosterBot.ts (hunk 2 above)
Comment (tsuwandy) [reply libraries/botbuilder/tests/teams/roster/src/rosterBot.ts]: yup!

**Comment by stevengum** [comment]
Closing this via https://github.com/microsoft/botbuilder-js/pull/1251.

**Pull request closed without merging**